src/app/modules/MainScreen/store/actions/index.ts

```diff
@@ -1,27 +1,42 @@
-import { ReturnType } from '@namespace/index';
+import { ReturnType, reduxAsyncAction } from '@namespace/index';
 
 export enum MainScreenTypes {
   INCREMENT = 'INCREMENT',
+  INCREMENT_SUCCESS = 'INCREMENT_SUCCESS',
```

Comment:
i'm wondering if that's not an overkill in this case. We're not fetching any sort of data, manipulate it at any point, so how possibly it could it fail?

Reply:
I know it's supposed to be a boilerplate for next project, but I think all developers will be familiar with the way failures/ successes should be handled